Src/Managers/theme_manager.py

```
@@ -2,135 +2,160 @@
-        cls.__update_item_theme(item_id)
+        self.__item_themes[item_id] |= set(theme_names)
+        self.__update_item_theme(item_id)
 
 
-    @classmethod
-    def remove_theme(cls, item_id: str | int, *theme_names: Themes):
+    def remove(self, item_id: str | int, *theme_names: Themes):
         """
         Удаляет указанные темы из списка тем элемента.
         args:
             item_id: str | int - идентификатор объекта, у которого удаляется тема
```

Ruff (0.15.21)

[warning] Docstring contains ambiguous `у` (CYRILLIC SMALL LETTER U). Did you mean `y` (LATIN SMALL LETTER Y)?

(RUF002)